### PR TITLE
Add more plugins to list of plugins

### PR DESCRIPTION
**Pull request by sergioramos**

### CODE REVIEW

File: doc/plugins.md [hunk 1]
```diff
@@ -166,8 +170,11 @@ See [Creating plugins][create] below.
     — make sure at most one top-level heading exists
 *   [`remark-numbered-footnote-labels`](https://github.com/jackfletch/remark-numbered-footnote-labels)
     — label footnotes with numbers
-*   [`remark-oembed`](https://github.com/agentofuser/remark-oembed)
+*   [`@agentofuser/remark-oembed`](https://github.com/agentofuser/remark-oembed)
     — transform URLs for youtube, twitter, etc. embeds
+*   [`remark-oembed`](https://github.com/sergioramos/remark-oembed)
+    — transform URLs surrounded by newlines into embeds that are loaded 
+    *asynchronously*
```
Comment (wooorm) [inline doc/plugins.md:177]:
```suggestion
    — transform URLs surrounded by newlines into *asynchronously* loaded embeds
```

A bit shorter, does that make sense to you?

File: doc/plugins.md [hunk 2]
```diff
@@ -177,6 +184,9 @@ See [Creating plugins][create] below.
 *   [`remark-ping`](https://github.com/zestedesavoir/zmarkdown/tree/HEAD/packages/remark-ping#readme)
     — custom syntax for mentions with configurable existence check (new node
     type, rehype compatible)
+*   [`remark-prism`](https://github.com/sergioramos/remark-prism)
+    — syntax highlight using [Prism](https://prismjs.com/) - with support for
+    most oficial plugins
```
Comment (wooorm) [inline doc/plugins.md:189]:
```suggestion
    — syntax highlight using [Prism](https://prismjs.com/) with support for
    most official plugins
```

Dash isn‘t really needed, and typo?

**Review by wooorm — CHANGES_REQUESTED** [review]
Thanks! Couple nits!

File: doc/plugins.md (hunk 1 above)
Comment (sergioramos) [reply doc/plugins.md]: It does make sense, thank you for the suggestion 👍

File: doc/plugins.md (hunk 2 above)
Comment (sergioramos) [reply doc/plugins.md]: good catch, thank you 👍

**Review by wooorm — APPROVED** [review]

**Comment by wooorm** [comment]
Thanksss!

**Pull request merged**